### PR TITLE
Adding basic Processing version of the JS QuadTree

### PR DESCRIPTION
**Pull request by Carla-de-Beer**
*version

### CODE REVIEW

File: Processing version/QTree/TheQuadTree.pde [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      } else if (this.southEast.insert(point)) {
+        return true;
+      } else if (this.southWest.insert(point)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  ArrayList<Point> query(Rectangle range, ArrayList<Point> found) {
```
Comment (Dezzles) [inline Processing version/QTree/TheQuadTree.pde]: Should this be List rather than ArrayList?

Comment (shiffman) [reply Processing version/QTree/TheQuadTree.pde]: `ArrayList` is good b/c it's what I use as standard catch-all flexible array in my tutorials and examples!

**Comment by WakeMeAtThree** [comment]
@shiffman I think making it part of a separate repo would work better, so that this repo keeps focus on it being JS library. We can then port examples to java as well to match the examples here. On the other hand, you did mention in a previous livestream that you were interested in creating some sort of processing library on eclipse CC. Can both things happen concurrently?

**Comment by shiffman** [comment]
Hi! I'm revisiting the QuadTree stuff as I'm teaching it in my class next week.  @Carla-de-Beer would you mind posting this code in your own repo and then adding a link to it from the README here?

**Comment by Carla-de-Beer** [comment]
Of course, yes. You are correct, it makes more sense to keep the P5,js repository clean. I added a new pull request (Update README.md #35) to allow for this. Apologies if I caused any inconvenience.

**Pull request closed without merging**